server-kotlin/src/main/kotlin/city_search/CitySearchController.kt

```diff
@@ -0,0 +1,19 @@
+package digital.cesko.city_search
+
+
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RestController
+import javax.websocket.server.PathParam
```

Comment:
is that really a websocket PathParam what we want there?

Reply:
Good catch